pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>tech.picnic.reactive-support</groupId>
     <artifactId>reactive-support</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>1.0</version>
```

Comment:
- Shall we go for [semantic versioning](http://semver.org/)?
- We should stick with snapshots. When we do a release a tag is created, and Jitpack will pick that up.
- Maybe start with 0.0.1? (But no strong preference.)